Remove check for install lib directory $libpath, because it does not exist yet
--- ni/src/ncl/MakeNcl.csh.orig	2019-02-28 00:44:39
+++ ni/src/ncl/MakeNcl.csh	2025-02-19 14:06:58
@@ -28,11 +28,6 @@
 set libmath       = "-lngmath"
 set ncarg_libs    = "$libncl $libnfp $libhlu $libncarg $libgks $libncarg_c $libmath"
 
-if (! -d "$libpath") then
-  echo "Library directory <$libpath> does not exist."
-  exit 1
-endif
-
 set files      = ""
 set extra_opts = ""
 
